diff -Naur WWW-Dilbert-1.06/lib/WWW/Dilbert.pm new/lib/WWW/Dilbert.pm
--- WWW-Dilbert-1.06/lib/WWW/Dilbert.pm	Mon Sep 20 00:44:25 2004
+++ new/lib/WWW/Dilbert.pm	Wed Sep 22 18:56:50 2004
@@ -2,6 +2,7 @@
 
 use strict;
 use warnings;
+use FileHandle ();
 use Image::Info ();
 use LWP::Simple ();
 use DBI ();
@@ -310,6 +311,22 @@
 	return $self->{strip_id};
 }
 
+sub write_to_file {
+	my $self = shift;
+	my $file_name = shift;
+	$file_name ||= $self->{strip_id} . '.' . $self->{image_info}->{file_ext};
+	my $file_handle = new FileHandle("> $file_name");
+	unless ($file_handle) {
+		die("Failed to open '$file_name' for writing:$!\n");
+	}
+	unless ($file_handle->print($self->{strip_blob})) {
+		die("Failed to write to '$file_name':$!\n");
+	}
+	unless ($file_handle->close()) {
+		die("Failed to close '$file_name':$!\n");
+	}
+}
+
 1;

